client/components/language-picker/index.jsx

```diff
@@ -135,9 +136,9 @@ export class LanguagePicker extends PureComponent {
 			return this.renderPlaceholder();
 		}
 
-		const [ langCode, langSubcode ] = language.langSlug.split( '-' );
-		const langName = language.name;
 		const { disabled, translate } = this.props;
+		const langName = language.name;
+		const langCodes = getLanguageCodeLabels( language.langSlug );
```

Comment:
We can destructure them here:
```
const { langCode, langSubcode } = ...
```
so that we don't need to access the same fields below for multiple times.